lib/juicebox/stream.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Juicebox.Stream do
+  use GenServer
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, [], name: JuiceboxStream)
+  end
+
+  def init([]) do
+    {:ok, []}
+  end
+
+  def get_queue_length do
```

Comment:
Is this a common pattern? i.e. to add helper methods to the module which then just do GenServer calls? I think if I was coming at this fresh I'd have 2 modules, one to be the GenServer and one to be the one that calls or casts to the gen server.